pkg/graveler/graveler_test.go

```diff
@@ -2292,3 +2292,182 @@ func TestGravelerCreateCommitRecord(t *testing.T) {
 		require.NoError(t, err)
 	})
 }
+
+func TestGraveler_Revert(t *testing.T) {
+	type fields struct {
+		CommittedManager *testutil.CommittedFake
+		RefManager       *testutil.RefsFake
+	}
+	type args struct {
+		ctx          context.Context
+		branchID     graveler.BranchID
+		ref          graveler.Ref
+		parentNumber int
+		allowEmpty   bool
+	}
+	tests := []struct {
+		name        string
+		fields      fields
+		args        args
```

Comment:
I think the test cases might be easier to read if we put `args` on the first line and `fields` on the second line.  That way when I read a test case, I know what to expect for each branch or commit name.

As an alternative, you could give some of the branches or commits a longer name -- I give one example below.